core/include/detray/masks/cylinder3.hpp

```diff
@@ -80,21 +80,17 @@ class cylinder3 final
     }
 
     /** Mask operation
-     *
-     * @tparam inside_local_type::point3 is the deduced type of the point to be
-     *checked w.r.t. to the mask bounds, it's assumed to be within the cylinder
-     *3D frame
      *
      * @param p the point to be checked
      * @param t is the tolerance tuple in (radius, z)
      *
      * @return an intersection status e_inside / e_outside
      **/
-    template <typename inside_local_t, bool is_rad_check = kRadialCheck>
+    template <typename cartesian_point_t>
```

Comment:
Would it make sense to remove this template and use __plugin::cartesian2 for point2 instead?